build.sbt

```diff
@@ -164,7 +207,39 @@ lazy val shadedAsm = project.
     autoScalaLibrary := false,
     exportJars := true,
     packageBin in Compile := (assembly in asmDeps).value,
-    releaseVersion := { ver => ver }
+    releaseVersion := { ver => ver },
+    bazelCustomBuild := BazelString(
+      """
+        |load(
+        |  '@io_bazel_rules_scala//scala:scala.bzl',
+        |  'scala_binary',
+        |  'scala_library',
+        |  'scala_test'
+        |)
+        |
+        |load(
+        |    "@com_github_johnynek_bazel_jar_jar//:jar_jar.bzl",
+        |    "jar_jar"
+        |)
+        |
+        |jar_jar(
```

Comment:
These two calls to jar_jar do the shading. Each is a separate target that rainier-core depends on up above.